pkg/spec/tf_job.go

```diff
@@ -11,20 +11,18 @@ import (
 )
 
 const (
-	TPRKind        = "tf-job"
-	TPRKindPlural  = "tfjobs"
-	TPRGroup       = "mlkube.io"
-	TPRVersion     = "v1beta1"
-	TPRDescription = "TensorFlow training job"
+	CRDKind        = "tf-job"
```

Comment:
Do you still use this anywhere? If this means Kind in the k8s API sense of Group-Version-Kind, then it should be camel case.